catalyst/rl/agents/layers.py

```diff
@@ -177,9 +177,9 @@ def _forward_lama(self, observation):
         x = self.head_net(x)
         return x
 
-    def forward(self, *args, with_log_pi=False, **kwargs):
+    def forward(self, *args, with_log_pi=False, deterministic=False, **kwargs):
```

Comment:
looks like we need to think about better forward API for agents... in future :)

Reply:
definitely